perceptual_filter/IndividualBehavior[] audience
string classificationType